app_dart/test/request_handlers/get_status_test.dart

```diff
@@ -115,11 +117,13 @@ void main() {
       final Commit commit1 = Commit(
           key: config.db.emptyKey.append(Commit,
               id: 'flutter/flutter/ea28a9c34dc701de891eaf74503ca4717019f829'),
-          timestamp: 3);
+          timestamp: 3,
+          branch: 'master');
       final Commit commit2 = Commit(
           key: config.db.emptyKey.append(Commit,
               id: 'flutter/flutter/d5b0b3c8d1c5fd89302089077ccabbcfaae045e4'),
-          timestamp: 1);
+          timestamp: 1,
+          branch: 'master');
```

Comment:
What about a test for when branch isn't given?

Reply:
Updated the `commit` model to have branch with default value `master.

Reply:
Does the datastore data have all tasks currently set to master if they're null? Otherwise wouldn't the model get overwritten from that null in datastore.

Reply:
I guess you meant `commit`, rather than `task`. Yes, if null, `master` is the default value now. (just pushed the changes)